lib/utils/compose_ts.ts

```diff
@@ -577,26 +578,42 @@ export async function validateProjectDirectory(
 	return result;
 }
 
-async function getTokenForPreviousRepos(
+async function pullServiceImages(
```

Comment:
This `pullServiceImages` function implements the suggested improvement in https://github.com/balena-io/balena-cli/issues/1045#issuecomment-620552222